app/views/users/_render.erb

```diff
@@ -0,0 +1,11 @@
+<% @users.each do |user| %>
+    <div class="render-container">
+        <div class="person">
+          <img class="img" src="<%= user.photo %>" alt="user-photo">
+        </div>
+        <div class="user-container">
+        <a href="<%= user_path(user.id) %>" class="username" ><%= user.name %></a>
```

Comment:
- [ ] [Optional] It would be better professionally if you used [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method instead of using the regular anchor tag `<a>`.